src/routes/connect/+page.svelte

```diff
@@ -41,7 +40,6 @@
       <Grid columns={6} columnGap="2rem" size="medium">
```

Comment:
```suggestion
      <Grid columns={5} columnGap="2rem" size="medium">
```

Reply:
Hmmm this is why you don't hardcode things 😁 If it were me I would have over-engineered it by making a list of icons and counting the length dynamically or something.

Reply:
Now it's all vertical on mobile...